### ISSUE TITLE
Definition of target pose

### ISSUE THREAD

**Issue by roboticswithjulia** [willshw/ArmVS#2]
Hello,
Regarding the following code: how could you know which is the target pose?  Do you create a specific frame for this target pose?

# set target pose
        _try:
            tf_listener.waitForTransform('/desired_camera_frame', '/tag_0', rospy.Time(), rospy.Duration(4.0))
            (t_target, R_target) = tf_listener.lookupTransform('/desired_camera_frame', '/tag_0', rospy.Time(0))
        except (tf.LookupException, tf.ConnectivityException, tf.ExtrapolationException):
            print 'Error! Cannot find [tag_0] to [desired_camera_frame] tf.'
            sys.exit(0)_

Regards and thanks,

Júlia

**Comment by willshw** [comment]
Yes, the target pose will be a frame that has a fixed transform to the tag.
